Documentation/rbac.md

```diff
@@ -34,7 +34,7 @@ for the different `ServiceAccounts` Rook uses to start the Rook Storage Pods.
 You need one `PodSecurityPolicy` that allows privileged `Pod` execution. Here is an example:
 
 ```yaml
-apiVersion: apps/v1
+apiVersion: policy/v1beta1
```

Comment:
The changes to rbac.md aren't technically related to the upgrade, right? This doc has been neglected, thanks for the fixes.

Reply:
I suppose they aren't directly related, except that I went through and verified upgrades with the PSP admission controller enabled and a fairly restrictive PSP enabled.